.changelog/unreleased/bug-fixes/524-rename-peerstate-tojson.md

```diff
@@ -0,0 +1,2 @@
+- [consensus] Rename (*PeerState).ToJSON to MarshalJSON to fix a logging data race
```

Comment:
```suggestion
- `[consensus]` Rename `(*PeerState).ToJSON` to `MarshalJSON` to fix a logging data race
```

Since this would otherwise constitute a dangling hyperlink in Markdown.